guides/WRITING-DOCUMENTATION.md

```diff
@@ -54,156 +66,165 @@ The description should describe what pattern does or should be used for in plain
 
 To extend the default functionality of KSS, we've implemented support for custom flags. Flags should always come after the description.
 
-Supported flags:
-
-- **`@hide-markup`** Hides the code snippet.
-- **`@react-props [NAME]`** Displays the React prop documentation for a component.
-
-  - **Example**: `@react-props Button.jsx`
-  - `[NAME]` is the filename of the react component where the `Proptypes` are defined.
-
-- **`@react-example [NAME]`** Displays the example file using this React component.
-
-  - **Example**: `@react-props Mask` displays `Mask.example.jsx`
-  - `[NAME]` is the filename of the react example to be displayed
-
-- **`@responsive`** Renders breakpoint toggles for the markup example.
-- **`@status [NAME]`** Displays a status badge. Supported values: `Draft`, `Work in progress`, `Ready`, `Deprecated`.
-- **`@uswds [URL]`** Marks the component as a US Web Design System component. Enter the URL so the documentation can link to the corresponding USWDS page.
+| flag name                   | example                                                      | flag description                                                                                                                        |
+| --------------------------- | ------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------- |
+| **`@hide-markup`**          | `@hide markup`                                               | Hides the code snippet.                                                                                                                 |
+| **`@react-props [NAME]`**   | `@react-props Button.jsx`                                    | Displays the React prop documentation for a component. `[NAME]` is the filename of the react component where the Proptypes are defined. |
+| **`@react-example [NAME]`** | `@react-example Button.example.jsx`                          | Displays the example file using this React component. `[NAME]` is the filename of the react example to be displayed.                    |
+| **`@responsive`**           | `@responsive`                                                | Renders breakpoint toggles for the markup example.                                                                                      |
+| **`@status [NAME]`**        | `@status Draft`                                              | Displays a status badge. `[NAME]` must be one of the following values: `Draft`, `Work in progress`, `Ready`, `Deprecated`.              |
+| **`@uswds [URL]`**          | `@uswds https://designsystem.digital.gov/components/button/` | `[URL]` is the link to the corresponding US Web Design System component.                                                                |
 
 ### Modifiers
 
```

Comment:
I believe we are in the process of deprecating modifiers. There are still pages that have it and our code still supports it, but maybe we should remove them from the documentation?

Reply:
Actually, we can remove it from the docs after its deprecated

Reply:
Yeah, I left it because it's still in use. I was thinking it could be worth revisiting as it's pretty powerful to show iterations of things without having to write a bunch of duplicate code.

Reply:
My understanding is that we moved away from that modifier model when we started to consolidate examples like with the button component. Modifiers can be convenient, but it also assumes the maintainer knows EJS (the templating language needed to use modifiers) and knows how the modifier generation works.

If we are eventually moving to markdown, this sort of organization wouldnt work as well long term, and i think its better to be explicit

Reply:
Yeah, that's a good point - I think we should remove the modifiers section from the docs then if we don't want folks using them.